sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlDslBase.java

```diff
@@ -70,7 +74,14 @@ public static void prepareClass() throws ParseException {
         Arrays.asList(Types.INTEGER, Types.BIGINT, Types.SMALLINT, Types.TINYINT, Types.FLOAT,
             Types.DOUBLE, Types.VARCHAR, Types.TIMESTAMP, Types.INTEGER, Types.DECIMAL));
 
+    rowTypeInTableB = BeamRecordSqlType.create(
```

Comment:
I think that using `BeamRecordSqlType.builder()` can help readability a little bit